ui/components/app/assets/defi-list/cells/defi-empty-state.tsx

```diff
@@ -1,45 +1,62 @@
-import React, { FC } from 'react';
```

Comment:
Updating DefiEmptyState component to incude locals, themed images and the TabEmptyState component